src/lib/loaders/loaders_without_authentication/gravity.js

```diff
@@ -11,9 +11,13 @@ export default requestIDs => {
     artistLoader: gravityLoader(id => `artist/${id}`),
     artistsLoader: gravityLoader("artists"),
     artworkLoader: gravityLoader(id => `artwork/${id}`),
+    fairLoader: gravityLoader(id => `fair/${id}`),
     fairsLoader: gravityLoader("fairs"),
+    filterArtworksLoader: gravityLoader("filter/artworks"),
+    geneArtistsLoader: gravityLoader(id => `gene/${id}/artists`),
     geneFamiliesLoader: gravityLoader("gene_families"),
     geneLoader: gravityLoader(id => `/gene/${id}`),
```

Comment:
This one already existed, so it’s apparently no problem, but unlike all others this has a leading slash.

Reply:
Oh yea, good point.